src/test/java/edu/harvard/iq/dataverse/DatasetFieldValueValidatorTest.java

```diff
@@ -111,6 +111,9 @@ public void testIsValid() {
         assertFalse(result);
     }
 
+    /* The next 7 tests below appear to duplicate tests in the ExternalIdentifierTest class.
+     * The ones here use the isValidAuthorIdentifier method which is only used in testing (and probabkly could be static).
+     */
```

Comment:
Yes, these tests duplicate those in the ExternalIdentifierTest and can probably be removed. The isValidAuthorIdentifier method is actually used in the DatasetAuthor class (around line 100).

Reply:
I think DatasetAuthor uses isValidIdentifier() without 'Author' . I'll go ahead and remove the tests.